src/mjolnir/bssbuilder.cc

```diff
@@ -579,21 +580,13 @@ void BssBuilder::Build(const boost::property_tree::ptree& pt,
                        const OSMData& osmdata,
                        const std::string& bss_nodes_bin) {
 
+  SCOPED_TIMER();
```

Comment:
why put this before the `if` here? certainly it doesnt make sense to time it if we are just going to return because we arent doing it?